mysql-test/suite/rocksdb_sys_vars/t/all_vars.test

```diff
@@ -1,3 +1,4 @@
+--source include/have_rocksdb.inc
```

Comment:
I was thinking the same thing when I added the test but really it seemed correct to not test existence of rocks tests if rocks is not an actual engine and just skip.